.k8s/K8s-Deployment.md

```diff
@@ -0,0 +1,45 @@
+# Kubernetes Deployment
+## Deployment
+### The Tool
+To work with a K8s cluster you need to install Kubectl, a tool for controlling your K8s deployment.
+[Kubectl installation guide](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+### Before Deploying
+Kubernetes manifests do not support adding config files directly in the manifest via a file path. Instead we need to use
```

Comment:
✏️ I would avoid using words like `we` or `our` and opt for something like `ConfigMaps are instead used to add configs.` And below this `ConfigMaps are used to pass configurations to the containers.`